File: app/src/main/java/io/github/zwieback/familyfinance/business/template/lifecycle/destroyer/TemplateViewDestroyer.java
package io.github.zwieback.familyfinance.business.template.lifecycle.destroyer;

import androidx.annotation.NonNull;

import java.sql.Connection;

import io.github.zwieback.familyfinance.core.lifecycle.destroyer.EntityViewDestroyer;

public class TemplateViewDestroyer extends EntityViewDestroyer {

    public TemplateViewDestroyer(Connection connection) {
        super(connection);
    }

    @NonNull
    @Override
    protected String getViewName() {
        return "v_template";
    }
}

File: app/src/main/java/io/github/zwieback/familyfinance/business/template/lifecycle/destroyer/TemplateViewDestroyer.kt
package io.github.zwieback.familyfinance.business.template.lifecycle.destroyer

import io.github.zwieback.familyfinance.core.lifecycle.destroyer.EntityViewDestroyer
import io.github.zwieback.familyfinance.core.model.constant.Views
import java.sql.Connection

class TemplateViewDestroyer(connection: Connection) : EntityViewDestroyer(connection) {

    override val viewName: String
        get() = Views.TEMPLATE
}
